opengrok-indexer/src/test/java/org/opengrok/indexer/history/FileHistoryCacheTest.java

```diff
@@ -607,6 +610,134 @@ public void testRenamedFilePlusChangesBranched() throws Exception {
                 updatedHistory.getHistoryEntries(), false);
     }
 
+
+    /**
+     * Make sure produces correct history where several files are renamed in a single commit.
+     */
+    @ConditionalRun(RepositoryInstalled.SubversionInstalled.class)
+    @Test
+    public void testMultipleRenamedFiles() throws Exception {
+        File reposRoot = new File(repositories.getSourceRoot(), "subversion");
+        History updatedHistory;
+
+        // The test expects support for renamed files.
+        RuntimeEnvironment.getInstance().setHandleHistoryOfRenamedFiles(true);
```

Comment:
set this to `false` at the end of the test ?

Reply:
Good point - there is already some resetting being done in the tearDown - I'll make sure that all variables are reset to their defaults. Unfortunately not a clean way to do this that I can see.